airflow/providers/samba/hooks/samba.py

```diff
@@ -16,12 +16,43 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import os
+from functools import wraps
+from logging import getLogger
+from shutil import copyfileobj
+from typing import Optional
 
-from smbclient import SambaClient
+from smbclient import (
+    getxattr,
+    link,
+    listdir,
+    listxattr,
+    lstat,
+    makedirs,
+    mkdir,
+    open_file,
+    readlink,
+    register_session,
+    remove,
+    removedirs,
+    removexattr,
+    rename,
+    replace,
+    rmdir,
+    scandir,
+    setxattr,
+    stat,
+    stat_volume,
+    symlink,
+    truncate,
+    unlink,
+    utime,
+    walk,
+)
 
 from airflow.hooks.base import BaseHook
 
+LOGGER = getLogger(__name__)
```

Comment:
`BaseHook` is a `LoggingMixin` so we can just do `self.log.info(...)` instead of instantiating a new module-level logger. Although style around this is pretty diverse in the current code base so I’m not quite sure if `LoggingMixin` is preferred or not…

Reply:
I actually didn't know hooks were `LoggingMixin`. Nice!